nginx/datadog_checks/nginx/nginx.py

```diff
@@ -48,14 +48,33 @@ def fromisoformat(ts):
     "stream/zone_sync": ["stream", "zone_sync"],
 }
 
+PLUS_API_V6_STREAM_ENDPOINTS = {
+    "stream/limit_conns": ["stream", "limit_conns"],
+}
+
+PLUS_API_V5_ENDPOINTS = {
+    "http/location_zones": ["location_zones"],
+    "resolvers": ["resolvers"],
+}
+
+PLUS_API_V6_ENDPOINTS = {
+    "http/limit_reqs": ["limit_reqs"],
+    "http/limit_conns": ["limit_conns"],
+}
+
```

Comment:
Would it be possible to merg all the `PLUS_API_V*` into a bigger struct? 
```python
PLUS_API_ENDPOINTS = {
  'v6': { .. }
  'v5': { .. }
}
```
It will be easier to define endpoints for new nginx plus versions

Reply:
Sure, makes sense!

Reply:
Updated to have two mappings, one for regular endpoints and one for stream endpoints